backend/apps/projects/models.py

```diff
@@ -11,17 +12,24 @@ class Project(models.Model):
     startup = models.ForeignKey(StartupProfile, on_delete=models.CASCADE)
     title = models.CharField(max_length=255)
     slug = models.SlugField(unique=True)
-    short_description = models.TextField()
+    short_desc = models.TextField()
     description = models.TextField()
     status = models.CharField(max_length=20, choices=STATUS_CHOICES)
     target_amount = models.DecimalField(max_digits=12, decimal_places=2)
     raised_amount = models.DecimalField(max_digits=12, decimal_places=2, default=0)
     currency = models.CharField(max_length=3, default="UAH")
+    thumbnail = models.URLField(blank=True, null=True)
     tags = models.TextField()
     visibility = models.CharField(max_length=20, default="public")
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
+    def clean(self):
+        if self.target_amount < 0:
+            raise ValidationError('Target amount cannot be negative.')
+        if self.raised_amount < 0:
+            raise ValidationError('Raised amount cannot be negative.')
```

Comment:
You implemented a clean() that raises ValidationError for negative amounts but didn't ensure it's called on save. Using Project.objects.create(...) bypasses clean() so invalid data can be persisted. Tests in this PR even create instances via objects.create.

Enforce validation by overriding save() to call full_clean() before saving, or add field-level validators on the DecimalFields. Example:
```
def save(self, *args, **kwargs):
    self.full_clean()
    super().save(*args, **kwargs)
```